libs/execution_context/ebpf_program.h

```diff
@@ -9,6 +9,8 @@
 #include "ebpf_program_types.h"
 #include "ebpf_protocol.h"
 
+#define EBPF_HASH_ALGORITHM "SHA256"
```

Comment:
Maybe add a comment saying the string must be one of those listed at https://learn.microsoft.com/en-us/windows/win32/seccng/cng-algorithm-identifiers

Reply:
Question: Can we add URLs in the comment?